torch/testing/_internal/dynamo_test_failures.py

```diff
@@ -2450,8 +2450,14 @@
     "TestMin.test_stack",  # functorch/test_dims
     "TestControlFlowTraced.test_tracing_map_symbolic_simple",  # functorch/test_control_flow
     "TestControlFlowTraced.test_nested_cond_map_cond_symbolic",  # functorch/test_control_flow
+    "TestControlFlowTraced.test_nested_map_cond_real",  # functorch/test_control_flow
```

Comment:
These expected failure tests are removed from previous PR that allows the tests to pass with graph breaks but graph breaks are not allowed after this PR so need to add them back.